meeting-notes/2019/Q1/2019-01-21--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,181 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 January 21, 2019
+
+- **Lead:** @alanshaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @jacobheun
+  - @vasco-santos
+  - @vmx
+  - @lidel
+  - @chriscool
+  - @mcollina
+  - @achingbrain
+- **Recording:** https://youtu.be/5Ka4b5WG2BM
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw
+- Done:
+    - Resolved issue with switch to bignumber.js ([#187](https://github.com/ipfs/js-ipfs-repo/pull/187)) to unblock [#1803](https://github.com/ipfs/js-ipfs/pull/1803) for 0.34 release
+    - Opened PR for disabling preload in test/CI environments [#1819](https://github.com/ipfs/js-ipfs/pull/1819)
+    - Fixed `hlsjs-ipfs-loader` for [`browser-video-streaming` example](https://github.com/moshisushi/hlsjs-ipfs-loader/pull/7)
```

Comment:
@jheisey has an video streaming example that doesn't require any external plugins. @jheisey can you make it a tutorial as well? it is a common request from our users :)